CrackingTheCodingInterview/Code/Chapter 2/Question2_8.cs

```diff
@@ -32,7 +32,47 @@ public static Node<T> FindLoopStart<T>(Node<T> head)
             return head;
         }
 
-        private class ReferenceComprer<T> : IEqualityComparer<T>
+        public static Node<T> FindLoopStartHareTortoise<T>(Node<T> head)
```

Comment:
Could you please add the following comments before this method (for consistency with the other methods in the solution):
```
// Space: O(1)
// Time: O(N)
```